statsmodels/graphics/gofplots.py

```diff
@@ -307,9 +357,21 @@ def qqplot(self, xlabel=None, ylabel=None, line=None, other=None,
             check_other = isinstance(other, ProbPlot)
             if not check_other:
                 other = ProbPlot(other)
-
-            fig, ax = _do_plot(other.sample_quantiles,
-                               self.sample_quantiles,
+            
+            s_self = self.sample_quantiles
+            s_other = other.sample_quantiles
+            
+            if len(s_self) > len(s_other):
+                raise ValueError("Sample size of `other` must be equal or " +
+                                 "larger than this `ProbPlot` instance")
```

Comment:
can we just swap s_self and s_other in this case?
I don't see an inherent difference between self and other, so we could just switch them. (Which is on the x and which is on the y axis might change dependent on relative size, does it? or is it easy to stick with `self` being always on the same plot axis/coordinate.